package.json

```diff
@@ -2,17 +2,21 @@
     "name": "@politie/informant",
     "version": "1.0.0-beta.0",
     "description": "Informant is a simple and fast logging library for Node.js and browser JavaScript applications.",
-    "main": "dist/index.js",
+    "main": "dist/informant.cjs.js",
```

Comment:
As we're also bundling all transpiled source files, I think having the rollup bundles in a folder `dist/bundles/<bundle>.<type>.js` might be a bit clearer.

Reply:
The bundles are the only JS files in the `dist` directory, so an additional directory seems like overkill to me.

Reply:
Note that only the three bundles are packed inside the NPM package.

Reply:
Fair enough. Didn't run `npm pack` yet. LGTM.